services/frontend-v3/Dockerfile

```diff
@@ -2,15 +2,21 @@ FROM node:12-alpine
 
 ENV TZ=America/Toronto
 
-# Copies code into container
+# Set working directory
 RUN mkdir /upskill-frontend
 WORKDIR /upskill-frontend
-COPY . .
+
+# Copy app dependencies and scripts
+COPY package*.json ./
+COPY ./scripts ./scripts/
```

Comment:
I'm curious as to why you are not copying the whole directory first

Reply:
i looked at bunch of documentation and they consistently do this. I tried looking around for a reason but i cant find any  explanation other than possible performance improvements. 

Here is one of many examples and a possible explanation ( I should add dockerignore file too):
![image](https://user-images.githubusercontent.com/11470442/89315772-3f3adb80-d649-11ea-89fe-d820d91bb901.png)

Let me know what you think

Reply:
I also found this:
![image](https://user-images.githubusercontent.com/11470442/89316647-6219bf80-d64a-11ea-9b6c-e5dd7baac9cc.png)

Reply:
Just saw that exact article haha, keep it as is, but can you do the same for the backend dockerfile then?

Reply:
thats a bit more difficult since you are using yarn generate which needs the database directory. Its 100% doable but it I will have to move stuff around a bit.

Reply:
mhhh you're right, then maybe we could just copy the src/database/ also